src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -277,8 +277,17 @@ public async Task InitializeAsync(IEnumerable<FunctionMetadata> functions, Cance
 
             var workerConfig = _workerConfigs.Where(c => c.Description.Language.Equals(_workerRuntime, StringComparison.InvariantCultureIgnoreCase)).FirstOrDefault();
 
+            // For other OOP workers, workerconfigs are present inside "workers" folder of host bin directory and is used to populate "_workerConfigs".
+            // For dotnet-isolated _workerConfigs is populated by reading workerconfig.json from the deployed payload(customer function app code).
+            // So if workerConfig is null and worker runtime is dotnet-isolated, that means isolated function code was not deployed yet.
+            var isDotNetIsolatedAppWithoutPayload = string.Equals(_workerRuntime, RpcWorkerConstants.DotNetIsolatedLanguageWorkerName, StringComparison.InvariantCultureIgnoreCase)
+                                                    && workerConfig == null;
```

Comment:
If the payload is not deployed, `_workerConfigs` will be empty (as we populate that from workerconfig,json of the deployed payload). So if `workerConfig` is null and worker run time is dotnet-isolated, we can infer that the payload was not deployed.

Reply:
What if a payload WAS deployed, but it is missing a worker config? Should we be treating that situation the same as no deployment at all?

Reply:
If the workerconfig file is not present, that should be considered as a non successful deployment.